docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -90,16 +99,18 @@ This method of provisioning the cluster would act as a single control point for
 - Create the new ClusterClass CRD which can serve as a collection of templates to create clusters.
 - Extend the Cluster object to use ClusterClass for creating managed topologies.
 - Enhance the Cluster object to act as a single point of control for the topology.
-- Extend the Cluster controller to create/update/delete managed topologies.
+- Extend the Cluster controller to create/update/delete managed topologies (this includes continuous reconciliation of the topology managed resources).
+- Introduce mechanisms to allow Cluster-specific customizations of a ClusterClass 
```

Comment:
In the Glossary, clusterClass
>A collection of templates that define a topology (control plane and machine deployments) to be used to create one or more clusters.

Should we replace with

>A collection of templates that define a topology (control plane and machine deployments) that continuously reconcile one or more clusters

or something similar?

Reply:
Done